### PR TITLE
Toggle enabled state of rules

### PR DESCRIPTION
**Pull request by paulsimonandrews**
https://eaflood.atlassian.net/browse/FCEP-68

Allow the web admin page to toggle the enabled state of action rules

### CODE REVIEW

File: server/views/actions.njk [hunk 1]
```diff
@@ -32,48 +33,66 @@
...
                       </dt>
-                      <dd class="govuk-summary-list__value">
+                      <dd class="govuk-summary-list__value" style="padding-left: 10px">
                         {{ rule.description }}
                       </dd>
+                      <dd class="govuk-summary-list__actions"></dd>
                     </div>
-                    <div class="govuk-summary-list__row">
-                      <dt class="govuk-summary-list__key">
+                    <div class="govuk-summary-list__row" style={{ backgroundColour|safe }}>
```
Comment (neilbmclaughlin) [inline server/views/actions.njk]: Given backgroundColour is assigned above not sure why we need `safe` here and in other usages of backgroundColour below.

**Review by neilbmclaughlin — COMMENTED** [review]
LGTM. 

Only issue I found was `enabling` or `disabling` does not use an anchor tag so the page load doesn't return the page to it's orginal position.

File: server/views/actions.njk (hunk 1 above)
Comment (paulsimonandrews) [reply server/views/actions.njk]: It seems Nunjucks (quite rightly) by default renders characters with special meaning in HTML using their entity characters (so `"` in the `backgroundColor` string gets replaced by `&quot;`). Using `safe` tells Nunjucks not to do this.

Comment (neilbmclaughlin) [reply server/views/actions.njk]: In which case would the following be preferable?

`{% set backgroundColour = 'background-color: white' %}`
and
` <div class="govuk-summary-list__row" style="{{ backgroundColour }}">`

Comment (paulsimonandrews) [reply server/views/actions.njk]: Absolutely, will change. Not sure why it wasn't originally 😕

**Review by neilbmclaughlin — DISMISSED** [review]

**Review by neilbmclaughlin — APPROVED** [review]
LGTM.

In the absence of client side AJAX calls I think the new workflow for enabling/disabling makes much more sense to me. 

Only issue is the diagnostics on the error page which we wouldn't want to leave in for anything public facing but is fine for the POC.

**Pull request merged**